cypress/support/routeSupport.js

```diff
@@ -37,7 +37,7 @@ Cypress.Commands.add("searchRouteWithAvoidTollOption", isResponsive => {
 	cy.wait(2000);
 	cy.get('[placeholder="To"]').click().type("manly beach sydney", { delay: 200 });
 	cy.contains("Beach").click({ force: true });
-	isResponsive ? cy.get('[data-testid="more-action-icon-container"]').click() : cy.contains("Route Options").click();
+	isResponsive ? cy.get('[data-testid="more-action-icon-container"]').click() : cy.contains("Avoid").click();
```

Comment:
Can the cypress identifier you are looking for be more specific to elements specific to AvoidToll or is this more than that?